plugins/topology/README.md

```diff
@@ -227,7 +248,7 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
 
 - The following label is added to the resources so that the Kubernetes plugin gets the Kubernetes resources from the requested entity:
 
-  ```yaml title="catalog-info.yaml"
+  ```yaml title="resource.yaml"
```

Comment:
This section coming below might confuse the user again, may we should have this before and at end add the details about label selector